### PR TITLE
Revision/69/anatomy

### PR DESCRIPTION
**Pull request by cliffle**

### CODE REVIEW

File: content/writing_tests/anatomy.md [hunk 1]
```diff
@@ -14,17 +14,49 @@ This guide assumes that you have finished the "Getting started" tutorial or that
...
+  - <i class="far fa-file"></i> **sakuli.properties** *(optional: Configuration for all  testsuites)*
 
 
 This file layout also represents the logical structure of Sakuli, which consists of a *testsuite* with one or more *testcase(s)*. Sakuli's `run` command takes the path to a testsuite folder and runs all testcases defined in `testsuite.suite` of the given folder. 
 
+### _logs
+
+The folder '_logs' is created automatically when Sakuli starts the first execution.
+To change the path of the created folder '_logs' you can use 'sakuli.log.folder'.
+See [Configuring Tests](/configuring_tests) for more information.
```
Comment (svettwer) [inline content/writing_tests/anatomy.md]:
```suggestion
The folder `_logs` is created automatically when Sakuli starts the first execution of a test suite.
To change the path of the created folder `_logs` you can use `sakuli.log.folder`.
See [Configuring Tests](/configuring_tests) for more information.
```

File: content/writing_tests/anatomy.md [hunk 2]
```diff
@@ -14,17 +14,49 @@ This guide assumes that you have finished the "Getting started" tutorial or that
...
-     - <i class="far fa-file"></i> **testsuite.suite** *Defines which testcases belong to the testsuite*
-     - <i class="far fa-file"></i> **testsuite.properties** *Configuration for the testsuite (overrides configuration from '../sakuli.properties')*
      - <i class="fas fa-folder"></i> **testcase** *Home of the testcase*
+         - <i class="far fa-file"></i> **.check.js.steps.cache** *An automatically generated cache file*
          - <i class="far fa-file"></i> **testcase.js** *The actual testcase*
+     - <i class="far fa-file"></i> **package.json** *Defines test script for container usage*
+     - <i class="far fa-file"></i> **testsuite.suite** *Defines which testcases belong to the testsuite*
+     - <i class="far fa-file"></i> **testsuite.properties** *Configuration for the testsuite (overrides configuration from '../sakuli.properties')*
+  - <i class="far fa-file"></i> **package.json** *All dependencies and setups for Node.js*
+  - <i class="far fa-file"></i> **sakuli.properties** *(optional: Configuration for all  testsuites)*
```
Comment (svettwer) [inline content/writing_tests/anatomy.md]:
```suggestion
  - <i class="far fa-file"></i> **sakuli.properties** *(optional) Configuration for all  testsuites*
```

File: content/writing_tests/anatomy.md [hunk 3]
```diff
@@ -14,17 +14,49 @@ This guide assumes that you have finished the "Getting started" tutorial or that
...
+  - <i class="fas fa-folder"></i> **_logs** *An automatically generated folder for error screenshots and sakuli.log*
+    - <i class="fas fa-folder"></i> **_screenshots** *Folder for error screenshots*
+    - <i class="far fa-file"></i> **sakuli.log** *Logs of Sakuli execution*
   - <i class="fas fa-folder"></i> **testsuite-a** *Home folder of a testsuite*
-     - <i class="far fa-file"></i> **testsuite.suite** *Defines which testcases belong to the testsuite*
-     - <i class="far fa-file"></i> **testsuite.properties** *Configuration for the testsuite (overrides configuration from '../sakuli.properties')*
      - <i class="fas fa-folder"></i> **testcase** *Home of the testcase*
+         - <i class="far fa-file"></i> **.check.js.steps.cache** *An automatically generated cache file*
          - <i class="far fa-file"></i> **testcase.js** *The actual testcase*
+     - <i class="far fa-file"></i> **package.json** *Defines test script for container usage*
```
Comment (svettwer) [inline content/writing_tests/anatomy.md]:
```suggestion
     - <i class="far fa-file"></i> **package.json** *(optional) Defines test script for container usage*
```

File: content/writing_tests/anatomy.md [hunk 4]
```diff
@@ -14,17 +14,49 @@ This guide assumes that you have finished the "Getting started" tutorial or that
 The minimum setup for a Sakuli project looks like this:
 
 - <i class="fas fa-folder"></i> **your-sakuli-project** *Home folder of the project*
-  - <i class="far fa-file"></i> **package.json** *All dependencies and setups for Node.js*
-  - <i class="far fa-file"></i> **sakuli.properties** *(optional: Configuration for all testsuites)*
+  - <i class="fas fa-folder"></i> **_logs** *An automatically generated folder for error screenshots and sakuli.log*
+    - <i class="fas fa-folder"></i> **_screenshots** *Folder for error screenshots*
+    - <i class="far fa-file"></i> **sakuli.log** *Logs of Sakuli execution*
```
Comment (svettwer) [inline content/writing_tests/anatomy.md]: Not indented correctly when website is build. `_screenshots` and `sakuli.log` should be elements of `_logs`.

image


In addition, `_logs` is created in every test suite not in the project itself.
image

File: content/writing_tests/anatomy.md [hunk 5]
```diff
@@ -14,17 +14,49 @@ This guide assumes that you have finished the "Getting started" tutorial or that
...
+- `sakuli.screenshot.onError`
+- `sakuli.screenshot.dir`
+- `sakuli.screenshot.storage`
+
+See [Configuring Tests](/configuring_tests) for more information.
+
+#### sakuli.log
+
+Sakuli logs all information about the test execution in this file. 
+To configure the log level you can use 'log.level'.
```
Comment (svettwer) [inline content/writing_tests/anatomy.md]:
```suggestion
To configure the log level, set `log.level` property in `sakuli.properties` or `testsuite.properties` accordingly. 
```

**Review by svettwer — CHANGES_REQUESTED** [review]

File: content/writing_tests/anatomy.md (hunk 4 above)
Comment (cliffle) [reply content/writing_tests/anatomy.md]: That's strange. For me, it was rendered correctly:
image

**Review by svettwer — APPROVED** [review]

**Pull request merged**